autoscale-workload-rabbit/src/main/java/com/hpe/caf/autoscale/workload/rabbit/RabbitWorkloadAnalyser.java

```diff
@@ -125,4 +144,18 @@ private ScalingAction getScalingAction(final ScalingOperation op, final int amou
             return ScalingAction.NO_ACTION;
         }
     }
+
+    @Override
+    public String retrieveEmailContent(final String percentageMem)
+    {
+        final String emailContenet
+            = "To whom it may concern, \n"
+            + "The RabbitMQ instance running on system " + System.getenv("CAF_RABBITMQ_MGMT_URL") + " is experiencing issues.\n"
+            + "\n"
+            + "RabbitMQ has used " + percentageMem + "% of its high watermark memory allowance.\n" 
+            + "\n"
+            + "From: Autoscaler\n";
+        return emailContenet;
```

Comment:
There's no need to read this into an variable - you can just use `return` directly.